skops/card/_model_card.py

```diff
@@ -205,6 +215,26 @@ def add_plot(self, **kwargs: str) -> "Card":
             self._figure_paths[plot_name] = plot_path
         return self
 
+    def evaluate(self, metric_values):
```

Comment:
I think `evaluate` implies there is some evaluation being done by this method, since it's a verb. What do you think of `add_metrics`